src/aboutus.html

```diff
@@ -6,22 +6,52 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Jura&display=swap" rel="stylesheet">
 	<title>About Us Page</title>
 </head>
-
-<body>
-	<div class="topnav">
-		<a href="index.html">home</a>
-		<a href="moodselector.html">mood selector</a>
-		<a href="results.html">your music</a>
-		<a class="active" href="aboutus.html">about us</a>
+<body class="aboutus">	
+	<nav>
+		<a href="index.html" class="reftoindex">home</a>
+		<a href="moodselector.html" class="reftomood">mood selector</a>
+		<a href="results.html" class="reftores">your music</a>
+		<a href="aboutus.html" class="reftoact">about us</a>
+	</nav>
+<div class = "projectbox-aboutus">
+	<h4 class = "project-aboutus">THE PROJECT</h4>
```

Comment:
Consider using an `h1` here instead, since this is the main page title - this is important for accessibility. If you want the size/styling of the h4 here, use CSS to accomplish that.